modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -126,6 +126,20 @@
 			<param index="0" name="value" type="Variant" />
 			<param index="1" name="type" type="Variant" />
 			<description>
+				Returns [code]true[/code] if [param value] is an instance of [param type]. The [param type] value must be one of the following:
```

Comment:
Given the recent bug reports we got, it seems like users use it this way:
```
var a = 5
var b = 42
print(is_instance_of(a, b))
```

Provided this does work, I don't think it's covered by the list below.

Reply:
It doesn't work because `42 >= Variant::VARIANT_MAX`.

https://github.com/godotengine/godot/blob/5f3dee5988391deffcc0de1039dd50e05e266913/modules/gdscript/gdscript_utility_functions.cpp#L530-L541

Reply:
I was thinking of #73663, but I guess this was just a bogus example? I thought they were trying to use this to confirm that both variants are ints.

Reply:
In the issue, I first thought the user was trying to use `is` like in Python. But maybe it really was just a bad example and the problem was the confusing error message.